internal/controller/ocivalidator_controller.go

```diff
@@ -53,18 +55,16 @@ func (r *OciValidatorReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 	validator := &v1alpha1.OciValidator{}
 	if err := r.Get(ctx, req.NamespacedName, validator); err != nil {
-		// Ignore not-found errors, since they can't be fixed by an immediate requeue
-		if apierrs.IsNotFound(err) {
-			return ctrl.Result{}, nil
+		if !apierrs.IsNotFound(err) {
```

Comment:
I took inspiration from flux [source controller](https://github.com/fluxcd/source-controller/blob/main/internal/controller/ocirepository_controller.go#L177) here.

I don't think a deleted CR will get caught in the reconciliation loop. Are you *certain* about the following?
>the plugin will always log errors whenever we delete a CR

Reply:
in testing it happened every time i deleted a CR

Reply:
In that case I need to revisit AWS & Network. Please do add the additional context though: `"key", req`

Reply:
would there be another reason for a not found error that we dont want to hide?